index.bs

```diff
@@ -974,10 +994,10 @@ optionally evidence of [=user consent=] to a specific transaction.
         account on a single authenticator. The platform is requested to return an error if the new credential would be created
         on an authenticator that also contains one of the credentials enumerated in this parameter.
 
-    :   <dfn>attachment</dfn>
-    ::  This member contains authenticator attachment descriptions, which are used as an additional constraint on which
-        authenticators are eligible to participate in a {{CredentialsContainer/create()}} or {{CredentialsContainer/get()}}
-        operation. See [[#attachment]] for a description of the attachment values and their meanings.
+    :   <dfn>authenticatorSelection</dfn>
+    ::  This member is intended for use by [=[RPS]=] that wish to select the appropriate authenticators to participate in 
+        the {{CredentialsContainer/create()}} or {{CredentialsContainer/get()}} operation based on their unique needs. 
+        See [[#Authenticator Selection Criteria]] for a description of all of the criterias that the [=[RPS]=] may impose. 
```

Comment:
This didn't link in https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/AngeloKai/webauthn/angelo-secondAuthnrSelect/index.bs#dom-makecredentialoptions-authenticatorselection. I think you can probably remove the entire sentence because it just links to the `AuthenticatorSelectionCriteria` type that's already linked by the definition of `authenticatorSelection`.